rotkehlchen/api/v1/schemas.py

```diff
@@ -275,38 +273,17 @@ def validate_order_by_schema(
             )
 
 
-class RequiredEvmAddressOptionalChainSchema(Schema):
-    address = EvmAddressField(required=True)
-    evm_chain = EvmChainNameField(
-        required=False,
-        limit_to=get_args(SUPPORTED_CHAIN_IDS),  # type: ignore
-        load_default=None,
-    )
-
-    @post_load
-    def transform_data(
-            self,
-            data: dict[str, Any],
-            **_kwargs: Any,
-    ) -> Any:
-        return EvmAccount(data['address'], chain_id=data['evm_chain'])
-
-
-class RequiredEvmlikeAddressOptionalChainSchema(Schema):
-    address = EvmAddressField(required=True)
-    chain = EvmChainLikeNameField(
-        required=False,
-        limit_to=SUPPORTED_EVM_EVMLIKE_CHAINS,  # type: ignore
-        load_default=None,
-    )
+class RequiredAddressOptionalChainSchema(Schema):
+    address = fields.String(required=True)
+    blockchain = BlockchainField(required=False, load_default=None)
```

Comment:
instead of doing `if account.blockchain in CHAINS_WITH_TRANSACTION` below, you can just limit this to allow blockchains with transactions.